prime-router/metadata/schemas/AZ/az-covid-19-hl7.schema

```diff
@@ -1,185 +1,13 @@
-
-  - name: standard.placer_name
-
-  - name: standard.placer_clia
-
-  - name: standard.placer_order_group_id
-
-  - name: standard.testing_lab_name
-
-  - name: standard.testing_lab_id
-
-  - name: standard.testing_lab_street
-
-  - name: standard.testing_lab_city
-
-  - name: standard.testing_lab_county_code
-
-  - name: standard.testing_lab_state
-
-  - name: standard.testing_lab_zip_code
-
-  - name: standard.patient_id
-
-  - name: standard.patient_id_type
-
-  - name: standard.patient_first_name
-
-  - name: standard.patient_last_name
-
-  - name: standard.patient_name_type_code
-
-  - name: standard.patient_middle_name
-
-  - name: standard.patient_suffix
-
-  - name: standard.patient_dob
-
-  - name: standard.patient_died
-
-  - name: standard.patient_gender
-    hl7Field: PID-8-1 # only write the code subcomponent
-
-  - name: standard.patient_race
-
-  - name: patient_race_system_version
-    default: 2.5.1
-    hl7Field: PID-10-7
-
-  - name: standard.patient_ethnicity
-
-  - name: standard.patient_street
-
-  - name: standard.patient_city
-
-  - name: standard.patient_state
-
-  - name: standard.patient_zip_code
-
-  - name: standard.patient_county_code
-
-  - name: standard.patient_phone_number
-
-  - name: standard.patient_age
-
-  - name: standard.pregnant
-
-  - name: standard.ordered_test_code
-
-  - name: standard.ordered_test_name
-
-  - name: standard.ordered_test_system
-
-  - name: standard.ordering_facility_name
-
-  - name: standard.ordering_facility_street
-
-  - name: standard.ordering_facility_city
-
-  - name: standard.ordering_facility_county_code
-
-  - name: standard.ordering_facility_state
-
-  - name: standard.ordering_facility_zip_code
-
-  - name: standard.ordering_facility_phone_number
-
-  - name: standard.ordering_provider_id
-
-  - name: standard.ordering_provider_id_authority
-
-  - name: standard.ordering_provider_id_authority_oid
-
-  - name: standard.ordering_provider_id_authority_type
-
-  - name: standard.ordering_provider_first_name
-
-  - name: standard.ordering_provider_last_name
-
-  - name: standard.ordering_provider_middle_name
-
-  - name: standard.ordering_provider_phone_number
-
-  - name: standard.testing_lab_specimen_id
-
-  - name: standard.resident_congregate_setting
-
-  - name: standard.specimen_collection_date_time
-
-  - name: standard.specimen_recieved_date_time
-
-  - name: standard.specimen_source_site_code
-
-  - name: standard.specimen_type
-
-  - name: standard.symptomatic_for_disease
-
-  - name: standard.test_kit_name_id
-
-  - name: standard.test_kit_name_id_type
-
-  - name: standard.test_performed_code
-
-  - name: standard.test_performed_name
-
-  - name: standard.test_performed_system
-
-  - name: standard.test_result
-
-  - name: standard.test_result_date
-
-  - name: standard.test_result_status
-
-  - name: standard.date_result_released
-
-  - name: standard.comment
+  - name: receiving_facility
+    default: AZDOH&2.16.840.1.114222.4.1.142&ISO
```

Comment:
With the `extends` a bunch of elements in the schema can just be the same as the base `covid-19` schema.

Reply:
And per my comment above, the covid-19 base schemas as about 1e6 fields in it.   We need a crisp way to say, out of all the zillion fields in my parent schema, these are the only ones to include in the child schema.

Put another way, covid-19.schema is really more of a data dictionary than it is a schema, then the children pick and choose which words from the dictionary they want to use.

Reply:
Right. The AZ HL7 implementation guide is pretty complete and their CSV request is minimal.